api/src/main.rs

```diff
@@ -0,0 +1,42 @@
+#[macro_use]
+extern crate rocket;
+use sqlx::postgres::PgPool;
+use sqlx::postgres::PgPoolOptions;
+
+#[get("/")]
+async fn index(pool: &rocket::State<PgPool>) -> String {
+    // Make a simple query to return the given parameter (use a question mark `?` instead of `$1` for MySQL)
+    let test_id: i32 = match sqlx::query!("SELECT id FROM test")
```

Comment:
Once you have structured data, you'll want to use sqlx's auto-destructuring into compile-checked types, check the documentation, it looks like this:

```rust
#[derive(sqlx::FromRow)]
struct User { name: String, id: i64 }

let mut stream = sqlx::query_as::<_, User>("SELECT * FROM users WHERE email = ? OR name = ?")
    .bind(user_email)
    .bind(user_name)
    .fetch(&mut conn);
```